department-of-veterans-affairs/va.gov-cms
# VACMS-7032 claro color regressions

## Description

Relates to #7032. issues surfaced from mob session:

- Dims the delete button color to not be so bright but still scary red.
- Fixes color contrast issues in the tab list as seen on `admin/content`
- Fixes the button display in the deploy form for claro, as seen on `admin/content/deploy`

## Testing done
See above

## Screenshots
<img width="801" alt="Screen Shot 2022-01-10 at 11 05 48 AM" src="https://user-images.githubusercontent.com/11279744/148825524-7dfbe5ca-574a-4411-9e96-6fd36dbef52b.png">
<img width="703" alt="Screen Shot 2022-01-10 at 11 13 25 AM" src="https://user-images.githubusercontent.com/11279744/148825528-433a7954-d297-44c8-bfa1-5651a0c9859f.png">
<img width="1783" alt="Screen Shot 2022-01-10 at 11 13 49 AM" src="https://user-images.githubusercontent.com/11279744/148825529-561b9e11-c391-4cfe-9446-a149ea815696.png">
<img width="1226" alt="Screen Shot 2022-01-10 at 11 13 59 AM" src="https://user-images.githubusercontent.com/11279744/148825532-8fa3e482-7af8-489e-b4b8-a02fd770e808.png">

## QA steps

As user that can deploy content,
1. go to admin/content and see the tab labels are easier to read when hovered and when in menu form
2. go to admin/content/deploy and see the button for viewing the front end is readable again
3. edit any piece of content, see that the delete button is still red, but not as eye-catching.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
